src/App/Fossa/Analyze.hs

```diff
@@ -280,7 +281,15 @@ data AnalyzeError
 instance Diag.ToDiagnostic AnalyzeError where
   renderDiagnostic :: AnalyzeError -> Doc ann
   renderDiagnostic ErrNoProjectsDiscovered =
-    "No analysis targets found in directory"
+    vsep
+      [ "No analysis targets found in directory."
+      , ""
+      , "Make sure your project meets the requirements described in:"
+      , "    " <> pretty userGuideUrl
+      , ""
+      , "If you believe this to be a defect, please file a request at:"
+      , "    " <> pretty newIssueUrl
```

Comment:
I would remove this. Most of our enterprise customers should be filing requests at `support@fossa.com`, and other users will know about our GitHub repo anyway (they had to install from it).